assets/js/data/cart/actions.js

```diff
@@ -206,6 +206,34 @@ export function* removeCoupon( couponCode ) {
 	return true;
 }
 
+/**
+ * Adds an item to the cart:
+ * - Calls API to add item.
+ * - If successful, yields action to add item from store.
+ * - If error, yields action to store error.
+ * - Sets cart item as pending while API request is in progress.
```

Comment:
This doesn't set the item as pending. Not sure if that's needed or not – either delete comment or add `itemQuantityPending()`s :)

Reply:
Probably not needed if this is used for All Products block only - unless the All Products block and Cart block are on the same page … 🤔